buildkite/src/Jobs/Test/VersionLint.dhall

```diff
@@ -35,8 +35,10 @@ let buildTestCmd
                 # RunInToolchain.runInToolchain
                     ([] : List Text)
                     "buildkite/scripts/version-linter.sh ${release_branch}"
-            , label = "Versioned type linter"
-            , key = "version-linter"
+            , label = "Versioned type linter for ${release_branch}"
+            , key = "version-linter-${release_branch}"
+            , if = Some
```

Comment:
Can you remove this and make them all soft-fail? I'll update the branch protection rules to make them required on the relevant branches then.

Reply:
yeah do that please.